packages/contracts/contracts/ConnextPriceOracle.sol

```diff
@@ -136,6 +142,12 @@ contract ConnextPriceOracle is PriceOracle {
         assetPrices[_token] = _price;
     }
 
+    function setV1PriceOracle(address _v1PriceOracle) external {
```

Comment:
If we're changing things, can we make the `require` a modifier? Is there a reason not to?

Reply:
In next line, `require(msg.sender == admin, "only admin can set v1PriceOracle address");` so what you want is to make this line as modifier???

Reply:
If it's used in other places, yeah i think it makes sense. right now it is used 5 times with the only major difference being the error messages.